### DISCUSSION TITLE
How to synchronize edits between two editor instances?

### DISCUSSION THREAD

**Discussion by punkpeye** [ueberdosis/tiptap#5829] [General Questions] [answered]
So my use case is that I allow talking with multiple AI agents at once.

<img width="1737" alt="Screenshot 2024-11-12 at 7 14 22 PM" src="https://github.com/user-attachments/assets/3d003b67-3836-41bc-a6f3-d67b156a34b4">

(screenshot https://glama.ai/ UI)

I am trying to find a way how I could synchronize user input across multiple editor instances.

**Comment by nperez0111** [comment, marked as answer]
Listen for the on update event and then take editor.getJSON() and set the other editor instances with editor.chain().setContent(json).run(). This would synchronize the states between the instances.
